Sources/muterCore/CLICommands/RunCommand/Steps/DiscoverMutationPoints.swift

```diff
@@ -48,18 +48,31 @@ private extension DiscoverMutationPoints {
         )
     }
     
-    func discoverNewMutationPoints(inFileAt path: String, containing source: SourceFileSyntax, configuration: MuterConfiguration) -> [MutationPoint] {
+    func discoverNewMutationPoints(
+        inFileAt path: String,
```

Comment:
As long as we're changing this, it could use the new `SourceFileInfo` type instead of these two parameters.

Reply:
oh, but the next parameter is a `SoruceFileSyntax`, not a `String`, so probably not.

Reply:
But then I found out about SourceCodeInfo. The plot thickens!